src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4629,6 +4629,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
                               both mean SARIF version 2.1.0.
 -reportanalyzer               Report additional analyzer information, such as
                               execution time.
+-analyzerconfig:&lt;file&gt;        Specify an EditorConfig file for analyzer                             
```

Comment:
This seems like an incomplete sentence. Consider something more like:
`Specific an EditorConfig file that controls specific diagnostics and analyzers.`

Reply:
man I wish there was an option in the compiler that can:
1. look for any *.ruleset file(s).
2. check if an .editorconfig file already exists.
3. if so check and see if that .editorconfig sets all the things in the *.ruleset file(s).
4. if not, merge what is not set in the .editorconfig from that *.ruleset file(s).
5. Delete the *.ruleset file(s).

I wish the compiler would do that and warn saying ``Found ruleset file, replacing it with .editorconfig if it already exists it will be appended to with anything that is in the ruleset that is not in the .editorconfig.`` Would make the move to .editorconfig a whole lot simpler in terms of people going as far as using vs to configure code styles and then dumping their global code styles on their machine to .editorconfig. So I think that the compiler would be best suited for the ``*.ruleset`` -> ``.editorconfig`` move.

Reply:
If this is a feature you want, consider filing a bug rather than commenting on a documentation pull request 😉